guides/Dataset.md

```diff
@@ -585,48 +694,384 @@ Or an open-ended date range _(thanks to [@lewismc](https://github.com/lewismc) f
+
+There are two main types of geologic time: Proper Intervals and Instants. They are diagrammed below and used in the examples that follow.
+
+  *Proper Interval*
+
+![Geologic Time 1](/assets/diagrams/dataset/dataset_temporal-coverage_geo-time_properInterval.svg "Dataset - Geologic Time - Proper Interval")
+
+  *Instant*
+
+![Geologic Time 2](/assets/diagrams/dataset/dataset_temporal-coverage_geo-time_instant.svg "Dataset - Geologic Time - Instant")
+
+These examples can be found in one JSON-LD file at [temporalCoverage.jsonld](/examples/dataset/temporalCoverage.jsonld)
+
+1. The dataset's temporalCoverage is described using ProperInterval, hasBeginning, and hasEnd elements from [OWL Time](http://www.w3.org/2006/time). The human readable description can be found in the description field: "Eruptive activity at Mt. St. Helens, Washington, March 1980 - January 1981". 
+
+  *Example*:
+
+<pre>
+{    "@context": {
+        "@vocab": "http://schema.org/",
+        "time": "http://www.w3.org/2006/time#",
+    },
+
+    "@type": "Dataset",
+        "description": "Eruptive activity at Mt. St. Helens, Washington, March 1980 - January 1981",
+<strong>        "temporalCoverage": ["1980-03-27T19:36:00/1981-01-03T00:00:00Z",
+            {
+                "@type": "time:ProperInterval",
+                "time:hasBeginning": {
+                     "@type": "time:Instant",
+                     "time:inXSDDateTimeStamp": "1980-03-27T19:36:00Z"
+                 },
+                 "time:hasEnd": {
+                    "@type": "time:Instant",
+                    "time:inXSDDateTimeStamp": "1981-01-03T00:00:00Z"
+                 }
+            }]
+}</strong>
+</pre>
+
+2. The dataset's temporalCoverage is described using Instant, inTimePosition, hasTRS, and numericPosition elements for a single geological date/age without uncertainties from [OWL Time](http://www.w3.org/2006/time).  Use a decimal value with appropriate timescale temporal reference system (TRS) and date/age unit abbreviation. Also provide a text form of the temporalCoverage (here "760 ka"). The human readable description can be found in the description field: "Eruption of Bishop Tuff, about 760,000 years ago". 
+
+   *Example*:
+
+<pre>
+{    "@context": {
+        "@vocab": "http://schema.org/",
+        "gsqtime": "https://vocabs.gsq.digital/object?uri=http://linked.data.gov.au/def/trs",
+        "time": "http://www.w3.org/2006/time#",
+        "xsd": "https://www.w3.org/TR/2004/REC-xmlschema-2-20041028/datatypes.html"
+    },
+    "@type": "Dataset",
+    "description": "Eruption of Bishop Tuff, about 760,000 years ago",
+    <strong>"temporalCoverage": ["760 ka",
```

Comment:
Same duplication issue as before, but also this string is not machine parseable following the xsd syntax.
```suggestion
    <strong>"temporalCoverage": [
```